pkgs/deb/debian/control

```diff
@@ -0,0 +1,56 @@
+Source: xnvme
+Section: devel
+Priority: optional
+Maintainer: Simon A. F. Lund <os@safl.dk>
+Build-Depends:
+ debhelper-compat (= 13),
+ libaio-dev,
+ liburing-dev,
+ meson (>= 0.58.0),
+Standards-Version: 4.6.2
+Homepage: https://xnvme.io/
+Vcs-Browser: https://github.com/OpenMPDK/xNVMe
+Vcs-Git: https://github.com/OpenMPDK/xNVMe.git
+Rules-Requires-Root: no
+
+Package: libxnvme0
```

Comment:
This package is called `xnvme` on all other distros, is there any reason to use a different name on debian?

Reply:
Yes, to comply with Debian policy.

Reply:
I see